src/applications/disability-benefits/all-claims/tests/components/ITFBanner.unit.spec.jsx

```diff
@@ -10,23 +10,33 @@ describe('ITFBanner', () => {
     expect(tree.text()).to.contain(
       'Your Intent to File request didn’t go through',
     );
+    tree.unmount();
   });
 
   it('should render an itf found message', () => {
     const tree = mount(<ITFBanner status="itf-found" />);
     expect(tree.text()).to.contain(
       'Our records show that you already have an Intent to File',
     );
+    tree.unmount();
   });
 
   it('should render an itf created message', () => {
     const tree = mount(<ITFBanner status="itf-created" />);
     expect(tree.text()).to.contain(
       'Thank you for submitting your Intent to File request',
     );
+    tree.unmount();
   });
 
   it('should throw an error', () => {
-    expect(() => mount(<ITFBanner status="nonsense" />)).to.throw();
```

Comment:
This is the only test I had to significantly refactor. comment explains what i did